packages/typespec-ts/test/modularUnit/scenarios.spec.ts

```diff
@@ -48,9 +48,9 @@ const OUTPUT_CODE_BLOCK_TYPES: Record<string, EmitterFunction> = {
   },
 
   // Snapshot of the entire models file
-  "(ts|typescript) models": async (tsp, {}, namedUnknownArgs) => {
-    const configs = namedUnknownArgs ? (namedUnknownArgs["configs"] as Record<string, string>): {};
-    const result = await emitModularModelsFromTypeSpec(tsp, configs);
+  "(ts|typescript) models": async (tsp, { }, namedUnknownArgs) => {
+    const configs = namedUnknownArgs ? (namedUnknownArgs["configs"] as Record<string, string>) : {};
+    const result = await emitModularModelsFromTypeSpec(tsp, { mustEmptyDiagnostic: false, ...configs });
```

Comment:
I wonder if we could add another set of test to allow mustEmptyDiagnostic: false, instead of changing the original test set?

Reply:
I may not prefer to have another set of test cases and for now I move this to markdown file to feed the config.